### PR TITLE
Using calculated scale instead of plain css

### PR DESCRIPTION
**Pull request by fedescarpa**
If CSS has obgect-fit contain the border image and the  border radius
does not adjust to image content.

### CODE REVIEW

File: lib/public/js/muzzle-editor.js [hunk 1]
```diff
@@ -45,8 +45,14 @@ $(() => {
   // Kids config
   // ===========
 
-  mumuki.kids.registerStateScaler(($state, fullMargin, preferredWidth, preferredHeight) => {
-    //  no manul image scalling. Leave it to css
+  mumuki.kids.registerStateScaler(($state, fullMargin) => {
+    const $image = $state.find('img');
+    if(!$image.length) return setTimeout(() => mumuki.kids.scaleState($state, fullMargin));
```
Comment (flbulgarelli) [inline lib/public/js/muzzle-editor.js]: Is this really necessary? Have we checked this situation is happening?

We had a lot of headaches with code like this in the past with gobstones runner, so I would like to ensure it is actually meaningful. 

Even if this code is necessary, shoudn't it be on the laboratory side? I think laboratory should ensure this code is called when possible, not even before that.

Comment (flbulgarelli) [reply lib/public/js/muzzle-editor.js]: @luchotc I'd like to hear your thoughts about my last statement.

Comment (luchotc) [reply lib/public/js/muzzle-editor.js]: Agree with you. @fedescarpa Do you find any case in which that actually happened or are we replicating gobstones code? I believe that it shouldn't be necessary nor here nor in gobstones. If it is, then labo is calling it when it shouldn't (but looking into the code it seems that is calling it after dom is loaded, so :man_shrugging:)

**Comment by luchotc** [comment]
Let's validate that the timeout is not necessary and integrate this one! @fedescarpa

**Pull request merged**